VideoLocker/build.gradle

```diff
@@ -234,6 +234,8 @@ android {
             if (fabric != null) {
                 ext.enableCrashlytics = fabric.get('FABRIC_KEY') != null
             }
+            def platformName = config.get('PLATFORM_NAME')
+            resValue "string", "app_name", platformName
```

Comment:
For some reason, the compiler seems to think that `platformName` is null, and fail with the error: "Build Config field cannot have a null parameter".